cli/cook/querying.py

```diff
@@ -8,6 +8,8 @@
 from operator import itemgetter
 from urllib.parse import urlparse, parse_qs
 
+import tenacity
```

Comment:
Are we worried that this might increase startup time?

Reply:
Changed it to a local import that only happens in the `--wait` case